enzy_htp/structure/structure.py

```diff
@@ -964,6 +965,22 @@ def build_metalcenters(self, dir, ft="PDB"):
 
     #endregion
 
+    def fix_pymol_naming(self) -> None:
+        """Fixes PyMOL's naming of atoms in-place using the PYMOL_TO_ATOM_MAPPER in chemical/residue.py.
+        Args:
+            stru: the Structure that contains the wrong atom names (if any).
+        Returns:
+            Nothing.
+        """
+        for chain in self.chains:
+            for residue in chain.residues:
+                for atom in residue.atoms:
+                    # special case; HA in GLY becomes HA2
+                    if residue.name == "GLY" and atom.name == "HA":
+                        atom.name = "HA2"
```

Comment:
Make sure the chirality is not messed up. Is HA2 always the [pro-S](https://chem.libretexts.org/Bookshelves/Organic_Chemistry/Book%3A_Organic_Chemistry_with_a_Biological_Emphasis_v2.0_(Soderberg)/03%3A_Conformations_and_Stereochemistry/3.12%3A_Prochirality) hydrogen.